### ISSUE TITLE
FPP Boot Delay: Optionally wait for valid date/time before fppd starts.

### ISSUE THREAD

**Issue by Jchuchla** [FalconChristmas/fpp#1669]
# Please answer the following:
**FPP Version and Hardware**: 
FPP 7.3.3 on Pi 3B+

**Describe the bug and Steps to reproduce**: 
On bootup, fpp will start schedule before NTP time sync is completed.  This results in starting the wrong schedule.
If I restart fppd after the time is synced, it jumps to the correct schedule.

**Expected behavior**: 
FPP should wait for time to sync before starting up the scheduler

**Comment by ghormann** [comment]
Closing.  Using fppd boot delay is the best solution.

**Comment by Jchuchla** [comment]
I do not agree.  Setting the startup delay is a workaround.  This is a repeatably failure on a common platform.  The default settings should either wait long enough, or actually test that the NTP sync has completed.

**Comment by Jchuchla** [comment]
@ghormann This ticket was prematurely closed.  It's not actually fixed.  Nor was enough information given on how to reasonably implement the workaround.

Can you point to where in the logs we can see the timestamps of where FPP startup is called, and where the NTP sync is logged.  This would point me in the direction of what value I can use for a startup delay.

**Comment by cpinkham** [comment]
I'm going from memory here, but we talked about this in the dev chat and I believe we decided that if we were to put any automatic NTP/time wait in, that it would be disabled by default so it wouldn't break existing systems out there.  We told people for years to create a 00:00:00-23:59:59 schedule entry to have FPP auto start a playlist without concern for the time.  We now have a FPP Command Preset which is a better way to do this, but there are lots of users still using the old method.  I think that the best way to implement this wait might be to add a special value flag to the boot delay setting.  If the delay is set to -1, startup would wait for the date to be >= 2024-01-01 00:00:00 which would indicate we should have a valid time from either the RTC or NTP.  This would be a simple while loop addition to the startup.  Grabbing the ticket and I'll take a look sometime.
